includes/class-woocommerce.php

```diff
@@ -368,6 +368,11 @@ public function includes() {
 		include_once WC_ABSPATH . 'includes/class-wc-auth.php';
 		include_once WC_ABSPATH . 'includes/class-wc-register-wp-admin-settings.php';
 
+		/**
+		 * Libraries
+		 */
+		include_once WC_ABSPATH . 'includes/libraries/action-scheduler/action-scheduler.php';
```

Comment:
Can we maybe have some sort of check before including since there are other plugins out there making use of this library already.

Reply:
@kloon it's better to always include this file to make sure if WC has a newer version of AS, that version is loaded. It's also safe to always load that file, because dependency management is built-into Action Scheduler.

Specifically, when a plugin loads the `action-scheduler.php` file, that instance of the Action Scheduler library will register itself and its version with a global `ActionScheduler_Versions` class (via `ActionScheduler_Versions::register()`). Then on `plugins_loaded` with priority `1`, after all plugins have been loaded. Action Scheduler will initialise only the newest version of itself found in the registry.

Reply:
OK, thanks for the explanation. Did a bit of further digging and see what you mean.